newDoc/api/model.md

```diff
@@ -1,15 +1,60 @@
+
 # class Model
 
-## static tableName
+> Subclasses of this class represent database tables.
+
+## Overview
+
+#### Model Lifecycle
+
+For the purposes of this explanation, let’s define three data layouts:
+
+1. `database`: The data layout returned by the database.
+1. `internal`: The data layout of a model instance.
+1. `external`: The data layout after calling model.toJSON().
+
+Whenever data is converted from one layout to another, converter methods are called:
+
+1. `database` -> [`$parseDatabaseJson`](/TODO/$parseDatabsaeJson) -> `internal`
+1. `internal` -> [`$formatDatabaseJson`](/TODO/$formatDatabaseJson) -> `database`
```

Comment:
These links can already be filled I think. We know where the functions will be. In this case /api/model.html#parsedatabasejson or simple #parsedatabasejson since we are in the same file.